documentation/source/rotating_logger.rst

```diff
@@ -0,0 +1,26 @@
+.. _rotating-logger:
+
+***********************
+Rotating Disk Log Files
+***********************
+
+The following is an example of how to support rotation of disk log files.
+It will keep the newest 3 files which are at a maximum 5 MiB large.
```

Comment:
```suggestion
It will keep the newest 3 files which are at a maximum of 5 MiB in size.
```

Reply:
I need to lose the "at" in the sentence too, right?

Reply:
Perhaps 'at most 5 MiB in size"

Reply:
Done.